lib/models/release.ts

```diff
@@ -389,6 +389,56 @@ const getReleaseModel = function (
 		);
 	}
 
+	/**
+	 * @summary Promotes a 'draft' release to 'final' (non-draft)
```

Comment:
I'm fine with it as is, but it felt like it's referring too much to the allowed values of the deprecated release_type field. How about simply something like v ?
```suggestion
	 * @summary Finalizes a draft release
```